portafly/src/i18n/locales/en/shared.yml

```diff
@@ -40,6 +40,8 @@ shared_elements:
   actions_header_description: Column listing the only available action to perform with each item of a table
   actions_header_plural: Actions
   actions_header_plural_description: Column listing the various available actions to perform with each item of a table
+  save_button: Save
```

Comment:
@porueesq did you refer to the Create button that submits the table with this string?

Reply:
Should it be `create_button` instead?

Reply:
Yes, @cntlsn ... I was testing the possibility of using `Save` in all creation pages. Though now I see it doesn't look good at all. I'll correct this.